src/protocols/hpr_protocol_router.erl

```diff
@@ -1,12 +1,89 @@
 -module(hpr_protocol_router).
 
+-include("grpc/autogen/server/packet_router_pb.hrl").
+-include("grpc/autogen/client/router_pb.hrl").
+
+% TODO: should be in a common include file
+-define(JOIN_REQUEST, 2#000).
```

Comment:
Just another heads-up: Anything using this constant should instead invoke library calls from https://github.com/helium/erlang-lorawan but can be resolved in a future PR.

@mikev is the primary author of that library and should be best for recommending what to use and/or introduce new functionality.  (n.b., there's also a companion Rust library in that repo but has seen fewer miles thus far.)